README.md

```diff
@@ -1,7 +1,18 @@
 Onboard-01.09.2022
 Projeto desenvolvido durante o processo de Onboarding
 Environment and Tools
--
+- Typescript
+- Android Studio Chipmunk 2021.2.1 Patch 2
+- Xcode 13.4.1 (13F100) - iOS 15.4 Simulator
+- Node 14.12.0
+- Pod 1.11.3
+
 Steps to run and debug
--
+- Open the terminal on the root of the project and run the command
+    npx react-native start
```

Comment:
nessas partes com código vc poderia incluir um [code block](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks)